### PR TITLE
Add position utility classes

### PR DESCRIPTION
**Pull request by Rleahy22**
## Description
@mpelzsherman brought up a good point about having position utility classes available: https://github.com/department-of-veterans-affairs/content-build/pull/505#discussion_r701202832

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Changes have been tested in vets-website
- [ ] Changes have been tested in IE11, if applicable
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: packages/formation/sass/core.scss [hunk 1]
```diff
@@ -2,82 +2,83 @@
...
+@import 'utilities/font-family';
+@import 'utilities/font-size';
+@import 'utilities/font-style';
+@import 'utilities/font-weight';
+@import 'utilities/height-width';
+@import 'utilities/line-height';
+@import 'utilities/margins';
+@import 'utilities/measure';
+@import 'utilities/padding';
+@import 'utilities/position';
```
Comment (Rleahy22) [inline packages/formation/sass/core.scss:81]: This is the only addition, the rest got picked up by prettier

File: packages/formation/sass/utilities/_position.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+// Responsive classes
+@each $bp_name, $bp_value in $breakpoints {
+  @media (min-width: $bp_value) {
+    @each $name in $position {
+      .#{$bp_name}\:vads-u-position--#{$name} {
+        position: #{$name} !important;
+      }
+    }
+  }
+}
```
Comment (bkjohnson) [inline packages/formation/sass/utilities/_position.scss:19]: Sass is fancy. Is this giving us the following:

```css
.vads-u-position--absolute {
    position: absolute !important;
}

.vads-u-position--fixed {
    position: fixed !important;
}

.vads-u-position--relative {
    position: relative !important;
}

@media (min-width: 320px) {
    .xsmall-screen:vads-u-position--absolute {
        position: absolute !important;
    }
}

// etc
```

**Review by bkjohnson — COMMENTED** [review]
Adding new utility classes means that we should probably have a [`position.md` document for them in here.](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/tree/master/src/_utilities)

@rtwell Are you ok with these new utility classes?

**Comment by Rleahy22** [comment]
@bkjohnson @rtwell Are we okay with these additions?

**Comment by rtwell** [comment]
if this is a) solving a known Design System problem, and b) implemented in a way that scales with the rest of the Design System, then yeah i don't see why we wouldn't add it.

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**